github/CodeScanAlert.py

```diff
@@ -19,140 +19,105 @@
 # along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
 #                                                                              #
 ################################################################################
+from __future__ import annotations
 
-from typing import Any, Dict
+from datetime import datetime
+from typing import Any
 
 import github.CodeScanAlertInstance
 import github.CodeScanRule
 import github.CodeScanTool
 import github.GithubObject
 import github.NamedUser
-import github.PaginatedList
+from github.GithubObject import Attribute, NonCompletableGithubObject, NotSet
+from github.PaginatedList import PaginatedList
 
 
-class CodeScanAlert(github.GithubObject.NonCompletableGithubObject):
+class CodeScanAlert(NonCompletableGithubObject):
     """
     This class represents alerts from code scanning.
     The reference can be found here https://docs.github.com/en/rest/reference/code-scanning.
     """
 
+    def _initAttributes(self) -> None:
+        self._number: Attribute[int] = NotSet
+        self._rule: Attribute[github.CodeScanRule.CodeScanRule] = NotSet
+        self._tool: Attribute[
+            github.CodeScanTool.CodeScanTool,
+        ] = NotSet
```

Comment:
Are these line breaks really needed?

Reply:
this formatted by black, not me

Reply:
Question is why, line 55 is much longer and not split.

Reply:
comma, I guess